engine/apps/schedules/models/custom_on_call_shift.py

```diff
@@ -173,6 +173,7 @@ class CustomOnCallShift(models.Model):
     )
     name = models.CharField(max_length=200)
     title = models.CharField(max_length=200, null=True, default=None)
+    # , choices = pytz.all_timezones
     time_zone = models.CharField(max_length=100, null=True, default=None)
```

Comment:
to remove

however, I'm wondering. Does it make more sense to add this validation @ the serializer level or model level?